Source/Controllers/AsyncMessagesViewController.swift

```diff
@@ -7,23 +7,28 @@
 //
 
 import Foundation
+import AsyncDisplayKit
+import SlackTextViewController
 
 class AsyncMessagesViewController: SLKTextViewController {
 
     let dataSource: AsyncMessagesCollectionViewDataSource
+    let asyncCollectionNode: ASCollectionNode
     override var collectionView: ASCollectionView {
         return scrollView as! ASCollectionView
     }
-
-    init(dataSource: AsyncMessagesCollectionViewDataSource) {
+    init?(dataSource: AsyncMessagesCollectionViewDataSource) {
         self.dataSource = dataSource
 
         let layout = UICollectionViewFlowLayout()
         layout.scrollDirection = UICollectionViewScrollDirection.vertical
-        let asyncCollectionView = ASCollectionView(frame: CGRect.zero, collectionViewLayout: layout)
+        
+        asyncCollectionNode = ASCollectionNode(frame: CGRect.zero, collectionViewLayout: layout)
```

Comment:
Nit: let's use the convenient `ASCollectionNode(collectionViewLayout)` initializer.

Reply:
Done